### DISCUSSION TITLE
How to update base model in Colab?

### DISCUSSION THREAD

**Discussion by luxerus** [lllyasviel/Fooocus#3814] [General]
When launching Focus cloning from the colab notebook, it loads an outdated Juggernaut XL base model version 8. What commands can be used to replace it with the latest version 11 from civitai? And the author would do well to update it in his repository. He hasn't updated it for a long time. I use what I have. But progress does not stand still. It's time to update the models. Thank you.

**Comment by Rogala** [comment]
[At the end of the Colab section, there's a mention of “Thanks to camenduru for the template!”
If you go to the git and search for the Fooocus section, you can find updated versions of the script.](https://github.com/lllyasviel/Fooocus?tab=readme-ov-file#colab)
